tests/Session/SessionTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Yiisoft\Yii\Web\Tests\Session;
+
+use Yiisoft\Yii\Web\Session\Session;
+use PHPUnit\Framework\TestCase;
+
+class SessionTest extends TestCase
+{
+    public function tearDown()
+    {
+        @session_destroy();
+    }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testGetAndSet()
+    {
+        $session = new Session();
+        $session->set('get', 'set');
+        self::assertEquals('set', $session->get('get'));
+    }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testHas()
+    {
+        $session = new Session();
+        $session->set('has', 'has');
+        self::assertTrue($session->has('has'));
+    }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testClose()
+    {
+        $session = new Session();
+        $session->set('close', 'close');
+        $session->close();
+        self::assertEquals(PHP_SESSION_NONE, session_status());
+        // because session_destroy() in tearDown doesn't work as expected
```

Comment:
Should we instead make it work as expected?

Reply:
Yes. I run tests without separated processes. With separating we don't need that and tearDown() function